### PR TITLE
Node 12 alpine

### PR DESCRIPTION
**Pull request by bikash-trussle**

### CODE REVIEW

File: build-images/alpine-node-12/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+# SonarQube
+RUN apk add --no-cache wget && \
+    wget https://dl.bintray.com/sonarsource/SonarQube/org/sonarsource/scanner/cli/sonar-scanner-cli/${SONAR_SCANNER_VERSION}/${SONAR_SCANNER_FILENAME} && \
+    unzip ${SONAR_SCANNER_FILENAME} && \
+    rm ${SONAR_SCANNER_FILENAME} && \
+    rm -rf ${SONAR_SCANNER_DIR}/conf && \
+    mv ${SONAR_SCANNER_DIR} ./sonar-scanner && \
+    mv sonar-scanner /root/ && \
+    apk del wget
```
Comment (Lngramos) [inline build-images/alpine-node-12/Dockerfile:47]: I don't believe we use SonarQube anymore. @danielingrey is that right?

File: build-images/alpine-node-12/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  py-pip \
+  nasm \
+  chromium-chromedriver \
+  chromium \
+  git
+
+# aws cli
+RUN pip install awscli
+
+# SonarQube
```
Comment (sergio-dias-trussle) [inline build-images/alpine-node-12/Dockerfile:39]: I thought that we no longer used SonarQube https://trussle.slack.com/archives/CQ4F3HDHQ/p1592495395345000

File: build-images/alpine-node-12/Dockerfile (hunk 1 above)
Comment (sergio-dias-trussle) [reply build-images/alpine-node-12/Dockerfile:47]: I asked about it recently, Vik said we don't: https://trussle.slack.com/archives/CQ4F3HDHQ/p1592495395345000

Comment (yannislionis) [reply build-images/alpine-node-12/Dockerfile:47]: Correct

**Comment by yannislionis** [comment]
Travis??

File: build-images/alpine-node-12/Dockerfile (hunk 1 above)
Comment (bikash-trussle) [reply build-images/alpine-node-12/Dockerfile]: cheers guys, removed

**Comment by bikash-trussle** [comment]
> Travis??



> Travis??

just realised 🤣 , i'll ignore this for the timebeing

**Pull request merged**